test/E2E/Helpers.purs

```diff
@@ -0,0 +1,365 @@
+module Test.E2E.Helpers
+  ( hasSelector
+  , startExample
+  , exampleUrl
+  , doJQ
+  , click
+  , checkSuccess
+  , enable
+  , setAttr
+  , buttonWithText
+  , password
+  , jqStr
+  , jqSet
+  , setText
+  , setValue
+  , trigger
+  , clickButton
+  , testPassword
+  , reactSetValue
+  , retrieveJQuery
+  , showOutput
+  , inWalletPage
+  , namiConfirmAccess
+  , namiSign
+  , geroConfirmAccess
+  , geroSign
+  , geroInit
+  , Selector(..)
+  , Action(..)
+  , NoShowPage(..)
+  , RunningExample(..)
+  , E2EOutput
+  ) where
+
+import Prelude
+
+import Control.Promise (Promise, toAffE)
+import Data.Array (head, filterA)
+import Data.Foldable (intercalate)
+import Data.Maybe (Maybe(Just, Nothing), maybe)
+import Data.Newtype (class Newtype, wrap, unwrap)
+import Data.Traversable (for, fold)
+import Effect (Effect)
+import Effect.Aff (Aff, launchAff_, delay)
+import Effect.Class (liftEffect)
+import Effect.Console (log)
+import Effect.Ref (Ref)
+import Effect.Ref as Ref
+import Effect.Uncurried (mkEffectFn1, EffectFn1)
+import Foreign (Foreign, unsafeFromForeign)
+import Test.E2E.Feedback (testFeedbackIsTrue)
+import Test.Spec.Assertions (shouldSatisfy)
+import Toppokki as Toki
```

Comment:
I don't think the few characters saved are worth it -- I'd recommend `as Toppokki` everywhere it's imported

Reply:
It's rather the difficulty of spelling it with 2 'p's and 2 'k's. If for one haven't even heard of the toppokki (the food) before and wouldn't know how to spell it. ;-)